### PR TITLE
Update to use tar.gz and use jujugui-1.8.1000. Does not install.

### PR DESCRIPTION
**Pull request by bac**

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -369,7 +379,7 @@ def test_no_releases(self):
     def test_no_releases_with_files(self):
         # A ValueError is raised if no releases are found.
         # Extraneous files are ignored while looking for releases.
-        self.add('jujugui-1.2.3.tgz')  # Wrong prefix.
+        self.add('hujugui-1.2.3.tgz')  # Wrong prefix.
```
Comment (urosj) [inline tests/test_utils.py]: hujugui?

Comment (urosj) [inline tests/test_utils.py]: Also, why 1.2.3?

File: tests/test_utils.py [hunk 2]
```diff
@@ -341,6 +341,16 @@ def test_xz(self):
...
 
+    def test_tar_gz(self):
+        # The last release is correctly retrieved for tar.gz files too.
+        self.add('juju-gui-0.12.1.tgz')
+        self.add('juju-gui-1.2.3.tgz')
+        self.add('juju-gui-2.0.0+build.42.tgz')
+        self.add('jujugui-2.0.1.tar.gz')
+        with self.mock_releases_dir():
+            path = get_release_file_path()
+        self.assert_path('jujugui-2.0.1.tar.gz', path)
```
Comment (urosj) [inline tests/test_utils.py]: Why do we have .tgz and .tar.gz?

File: tests/test_utils.py (hunk 1 above)
Comment (bac) [inline tests/test_utils.py]: it is just test data.  4.5.6 better?  :)

File: tests/test_utils.py (hunk 2 above)
Comment (bac) [inline tests/test_utils.py]: The old way used .tgz.  The 'make dist' target of juju-gui now produces jujugui-VERSION.tar.gz.  I figured it was easier to adapt the charm to what was being produced than to either update juju-gui or require a manual renaming when copying to the charm.

I see no problem with keeping tgz and xz.

File: tests/test_utils.py (hunk 1 above)
Comment (urosj) [inline tests/test_utils.py]: Hah, note to self, don't review in the evening :)

**Comment by urosj** [comment]
:+1:

**Comment by jaycee** [comment]
:+1:

**Comment by bac** [comment]
Thanks for the reviews.

**Pull request merged**